### ISSUE TITLE
Wrong filter name in method "content" of the Comment class 

### ISSUE THREAD

**Issue by seregahere** [timber/timber#1358]
### Expected behavior
The method "content" of the "Comment" class should return the comment content with filter 'comment_text' applied.

### Actual behavior
The method "content" applies the wrong filter: 'get_comment_text '. Also, space should not present in the filter name. The right filter name: 'comment_text'. Reference: [https://developer.wordpress.org/reference/hooks/comment_text/](url)

### Suggestion
I suggest the following patch:
```javascript
--- lib/Comment.php	2017-03-16 11:58:31.045474652 +0300
+++ lib/Comment-new.php	2017-03-16 12:00:50.211051758 +0300
@@ -155,7 +155,7 @@
 	 * @return string
 	 */
 	public function content() {
-		return apply_filters('get_comment_text ', $this->comment_content);
+		return apply_filters('comment_text', $this->comment_content);
 	}
 
 	/**
```

**Comment by jarednova** [comment]
Thanks @seregahere! I can make the fix, but do you want to submit a PR so you get commit credit?

**Comment by seregahere** [comment]
@jarednova , just make the fix :) Thanks!

**Comment by jarednova** [comment]
well I'll be thinking of you while I do it — thanks again for the issue and the fix!

**Comment by jarednova** [comment]
closed by 0e11f53641f458bac517572d1fc28e2b4137d261
